Src/Amr/AMReX_Amr.cpp

```diff
@@ -1930,6 +1933,8 @@ Amr::timeStep (int  level,
     // Update so that by default, we don't force a post-step regrid.
     amr_level[level]->setPostStepRegrid(0);
 
+    if(max_level==0 && force_regrid_level_zero)regrid_level_0_on_restart();
```

Comment:
We are about to enforce https://clang.llvm.org/extra/clang-tidy/checks/readability/braces-around-statements.html